site/app/libraries/database/DatabaseQueries.php

```diff
@@ -4873,6 +4873,21 @@ public function getInstructorLevelUnarchivedCourses(string $user_id): array {
         return $this->submitty_db->rows();
     }
 
+    /**
+     * Get an array of items with each item including the id, type, text, and date.
+     *
+     * @return array<array{id: int, type: int, text: string, date: string}> An array of items.
+     */
+    public function getGlobalCalendarItems(): array {
+        $query = "
+            SELECT id, type, text, date
+            FROM global_calendar_items
+            ORDER BY date DESC
+        ";
+        $this->submitty_db->query($query);
+        return $this->submitty_db->rows();
+    }
```

Comment:
You don't need to make new queries when using Doctrine. Instead fetch them with Doctrine utilities. See other controllers that use doctrine entities for examples.